From: qtonthat <qtonthat@02d69b8e-d120-0410-893f-fe71a3b6e39e>
Date: Tue, 22 Jul 2008 14:21:44 +0000 (+0000)
Subject: Google maps no longer worked due to changes by Google. Problems now fixed.
X-Git-Url: http://repo.or.cz/w/viking.git?a=commitdiff_plain;h=47d2edf771db3a0c040e5fb042ca87ab68cc0e10

Google maps no longer worked due to changes by Google. Problems now fixed.

Signed-off-by: Quy Tonthat <qtonthat@gmail.com>

Conflicts:

Signed-off-by: Quy Tonthat <qtonthat@gmail.com>


git-svn-id: https://viking.svn.sourceforge.net/svnroot/viking/trunk@733 02d69b8e-d120-0410-893f-fe71a3b6e39e
---

diff --git a/ChangeLog b/ChangeLog
index 34164fe..119ed06 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,8 @@
+2008-07-22
+Quy Tonthat <qtonthat@gmail.com>:
+	* Google maps no longer worked due to changes made by Google.
+	  The Problems now fixed.
+
 2008-07-21:
 Guilhem Bonnefille <guilhem.bonnefille@gmail.com>:
 	* Release Viking 0.9.5
diff --git a/src/google.c b/src/google.c
index c56b7d9..bb5d370 100644
--- a/src/google.c
+++ b/src/google.c
@@ -117,8 +117,8 @@ static const gchar *google_version_number(MapCoord *mapcoord, GoogleType google_
   gsize len;
   gchar *gvers, *tvers, *kvers, *terrvers, *tmpvers;
   static DownloadOptions dl_options = { "http://maps.google.com/", 0, a_check_map_file };
-  static const char *gvers_pat = "http://mt0.google.com/mt?n\\x3d404\\x26v\\x3d";
-  static const char *kvers_pat = "http://khm0.google.com/kh?n\\x3d404\\x26v\\x3d";
+  static const char *gvers_pat = "http://mt0.google.com/mt?v\\x3d";
+  static const char *kvers_pat = "http://khm0.google.com/kh?v\\x3d";
 
   g_assert(google_type < TYPE_GOOGLE_NUM);
 
